miniapps/shifted/algoim_test.cpp

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
We should add the standard copyright stuff and add a description of the miniapp here. 
We should also probably call the miniapp something else instead of `algoim_test`.. maybe something like `cut-integral`.

Reply:
changed to lsf_integral